packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -232,9 +252,31 @@ export const Stepper = React.forwardRef<HTMLDivElement, StepperProps>(
               ? StepStatus.active
               : StepStatus.incomplete;
 
+          const bothLabels = stepLabels.some(child => {
+            if (React.isValidElement(child)) {
+              return (
+                Object.keys(child.props).includes('label') &&
+                Object.keys(child.props).includes('secondaryLabel')
+              );
+            }
+          });
+
+          const label = stepLabels.some(child => {
+            if (React.isValidElement(child)) {
+              return Object.keys(child.props).includes('label');
+            }
+          });
+
+          const secondaryLabel = stepLabels.some(child => {
+            if (React.isValidElement(child)) {
+              return Object.keys(child.props).includes('secondaryLabel');
+            }
+          });
```

Comment:
Right now, this is getting evaluated inside the loop of children, so this gets called a lot of times. We actually only need to call this once, so here are a few suggestions on how we can make this more efficient and concise:
- move this outside of the loop
- update variable names to be descriptive-- what we are doing here is checking if all the children have labels, so let's call it that for clarity. don't be afraid to add comments if you think any logic could be confusing in the future
- since label & secondaryLabel are using the same loop, we can combine those checks for efficiency
- we don't need an additional `bothLabels` check if we already have a value for label and secondarylabel, we can just check if both are true
- our ternary was getting a little crazy, so we could simplify that as well

Here's an example: 
![image](https://github.com/cengage/react-magma/assets/91160746/073a11f9-19e4-419f-9571-fb0b727957b1)